docs/overview/trustbloc-did-method.md

```diff
@@ -128,6 +174,18 @@ This 64 bit unsigned integer element specifies the number of stakeholders that a
 
 If this element is not present in the consortium policy, the default value of `num-queries` is the number of stakeholders within the consortium.
 
+##### Sidetree Parameters
+`"Sidetree": {[parameters]}`
+
+This object holds parameters which the client needs for Sidetree requests. The keys it must contain, with value types and example values, are as follows:
+
+**Key** | **Value Type** | **Description** | **Example Value**
+--- | --- | --- | ---
+`"hash-algorithm"` | `string` | The hash algorithm used for Sidetree operation requests |  `"SHA256"`
+`"key-algorithm"` | `string` | The key algorithm used for signing Sidetree operation requests | `ES256`
+`max-encoded-hash-length` | `uint64` | The maximum string length of the hash created for the operation request | `100`
+`"max-operation-size"` | `uint64` | The maximum size of the Sidetree operation request, in bytes | `8192`
+
```

Comment:
Should check with Bob and Sandra for additional parameters that are needed to validate the backed data structures via a block and DCAS rest interface.